### PR TITLE
Feature/upgrade version

### PR DESCRIPTION
**Pull request by ganievs**
Upgrade module versions
Make terraform v0.12 compatible

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,48 +1,38 @@
...
 }
 
 module "aws-cert" {
-  source = "github.com/lean-delivery/tf-module-aws-acm"
-
-  providers = {
-    aws = "aws.acm_provider"
-  }
+  source  = "terraform-aws-modules/acm/aws"
+  version = "~> v2.0"
```
Comment (ToROxI) [inline main.tf:12]: Please update syntax according 0.12 specification.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
 # tf-module-aws-s3-cf-acm
 Terraform module for static site with SSL
+
```
Comment (ToROxI) [inline README.md:4]: Please declare required Terraform version for this module (or module version).

File: main.tf [hunk 3]
```diff
@@ -1,48 +1,38 @@
...
+  domain   = "${var.domain}"
+  statuses = ["ISSUED", "PENDING_VALIDATION"]
   provider = "aws.acm_provider"
+
+  depends_on = [module.aws-cert]
 }
 
 module "cdn" {
-  source                       = "git::https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn.git?ref=cfa99645ac8a949b67d32d302fdf4f1996d685e9"
+  source                       = "git::https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn.git?ref=0.11.0"
```
Comment (ToROxI) [inline main.tf]: Version 0.11.0 is for Terraform 0.12, as far as I see.

File: main.tf (hunk 1 above)
Comment (ganievs) [reply main.tf]: From module readme - Terraform 0.12. Pin module version to ~> v2.0

File: variables.tf [hunk 4]
```diff
@@ -1,80 +1,74 @@
...
-  type        = "map"
+  type        = map(string)
   default     = {}
   description = "Additional tags (e.g. map(`BusinessUnit`,`XYZ`)"
 }
 
 variable "enabled" {
   default     = "true"
-  type        = "string"
+  type        = string
```
Comment (ToROxI) [inline variables.tf]: type should be bool, i suppose

File: variables.tf [hunk 5]
```diff
@@ -1,80 +1,74 @@
...
 variable "acm_certificate_arn" {
   description = "Existing ACM Certificate ARN"
-  type        = "string"
+  type        = string
   default     = ""
 }
 
 variable "use_regional_s3_endpoint" {
-  type        = "string"
+  type        = string
```
Comment (ToROxI) [inline variables.tf]: type should be bool, i suppose

**Review by ToROxI — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 5 above)
Comment (ganievs) [reply variables.tf]: fix

**Review by ToROxI — APPROVED** [review]

**Pull request merged**